lava-v2-jobs-from-api.py

```diff
@@ -36,19 +36,16 @@ def main(args):
     config = configuration.get_config(args)
     plans = config.get("plans")
     targets = config.get("targets")
-    if config.get("jobs"):
-        job_dir = setup_job_dir(config.get("jobs"))
-    else:
-        job_dir = setup_job_dir(os.getcwd() + '/jobs')
+    api = config.get('api') or "https://api.kernelci.org"
```

Comment:
Lets just remove the defaults, it's probably preferable to let the script complain when it cant find a value for api, storage or token